src/components/Guest.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from "react";
+import styled from "@emotion/styled";
+
+const StyledGuest = styled.div`
+  position: relative;
+  display: block;
+  letter-spacing: 0.05em;
+  background-color: rgba(240, 240, 245, 0.7);
+  color: rgba(54, 63, 84, 0.7);
+  border-radius: 0.25em;
+  padding: ${props =>
+    props.removed
+      ? "0 1.8em 0 0.8em"
+      : props.mounted
+      ? "0 1.8em 0 0.8em"
+      : "0 1.8em 0 0.8em"};
+  margin-bottom: ${props =>
+    props.removed ? "0" : props.mounted ? "0.5em" : "0"};
+  width: max-content;
```

Comment:
`max-content` is not supported by Internet Explorer or Microsoft Edge. Would this work for all browsers if `display` is set to `table`?

Plus, this is the recommended usage of the `max-content` value via Firefox documentation:

```css
width: intrinsic;           /* Safari/WebKit uses a non-standard name */
width: -moz-max-content;    /* Firefox/Gecko */
width: -webkit-max-content; /* Chrome */
```

https://developer.mozilla.org/en-US/docs/Web/CSS/width#max-content_2